selfswab/tests/test_tasks.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+import responses
+from django.test import TestCase, override_settings
+from django.utils import timezone
+from selfswab.models import SelfSwabTest
+from selfswab.tasks import poll_meditech_api_for_results
+
+
+class PollMeditechForResults(TestCase):
+    updated_at = timezone.now()
+
+    flow_response = {
+        "uuid": "93a624ad-5440-415e-b49f-17bf42754acb",
+        "flow": {
+            "uuid": "f5901b62-ba76-4003-9c62-72fdacc1b7b7",
+            "name": "SelfSwabTest",
+        },
+        "groups": [
+            {"uuid": "04a4752b-0f49-480e-ae60-3a3f2bea485c", "name": "SelfSwabStudy"}
+        ],
+        "contacts": [{"uuid": "5079cb96-a1d8-4f47-8c87-d8c7bb6ddab9", "name": "Joe"}],
+        "restart_participants": True,
+        "status": "pending",
+        "created_on": "2015-08-26T10:04:09.737686+00:00",
+        "modified_on": "2015-09-26T10:04:09.737686+00:00",
+        "extra": {"result": "Negative", "updated_at": updated_at.strftime("%d/%m/%Y")},
+    }
+
+    @responses.activate
+    @override_settings(
+        RAPIDPRO_URL="https://rp-test.com",
+        MEDITECH_URL="https://medi-test.com",
+        SELFSWAB_RAPIDPRO_TOKEN="123",
+        SELFSWAB_RAPIDPRO_FLOW="321",
+    )
+    def test_poll_to_meditch_for_results(self):
+        """
+        Should sync a barcode and get results from api,
+        and save result
+        """
+        SelfSwabTest.objects.create(
```

Comment:
Change these two to "Pending" and add another one that is Positive, to make sure the positive one is not sent to the meditech api

Reply:
done